commit 4b8ef6bed62daced90c7826985650c2a813d2996
Author: Jonathan Riddell <jr@jriddell.org>
Date:   Wed Feb 8 14:56:48 2017 +0000

    remove modern C++ use to fix compile with current KDE policy

diff --git a/src/backends/python/pythonhighlighter.cpp b/src/backends/python/pythonhighlighter.cpp
index 4064524..87b10dd 100644
--- a/src/backends/python/pythonhighlighter.cpp
+++ b/src/backends/python/pythonhighlighter.cpp
@@ -87,7 +87,7 @@ void PythonHighlighter::highlightBlock(const QString &text)
     while (pos < text.length()) {
         // Trying to close current environments
         bool triggered = false;
-        for (int i = 0; i < flags.size() and not triggered; i++) {
+        for (int i = 0; i < flags.size() && !triggered; i++) {
             int flag = flags[i];
             QRegExp &regexp = regexps[i];
             QTextCharFormat &format = formats[i];
@@ -126,7 +126,7 @@ void PythonHighlighter::highlightBlock(const QString &text)
         singleLineCommentStart.indexIn(text, pos);
 
         if (singleLineCommentStartPos != -1
-            and singleLineCommentStartPos < minPos) {
+            && singleLineCommentStartPos < minPos) {
             setFormat(pos, text.length() - pos, commentFormat());
         break;
             } else if (minRegexp) {
